content/efcore/getting-started/loading-related-data.md

```diff
@@ -24,67 +24,67 @@ using (var context = new LibraryContext())
 { 
 	var book = context.Books  
 		.Include(b => b.Author)
-		.Single(b => b.BookId == 1); 
+		.Single(b => b.Id == 1); 
 } 
 ``` 
 :::repl{data-name=include-method} 
 :::
  
-You can also include multiple non-primitive type properties at once by calling the `Include` method multiple times. For example, we can include the `Editions` and `Authors` properties of the `Book` entity like so:  
+You can also include multiple non-primitive type properties at once by calling the `Include` method multiple times. For example, we can include the `CheckoutRecords` and `Author` properties of the `Book` entity like so:  
  
 ```{.snippet} 
 using (var context = new LibraryContext()) 
 { 
 	var book = context.Books 
-		.Include(b => b.Editions) 
+		.Include(b => b.CheckoutRecords) 
 		.Include(b => b.Author)
-		.Single(b => b.BookId == 1); 
+		.Single(b => b.Id == 1); 
 } 
 ``` 
 :::repl{data-name=multiple-include-methods} 
 :::
  
 ## Multi-layer Inclusion 
  
-Now, what happens if one of the properties we include also has non-primitive type properties? By default, they won't be loaded from the database, but we can tell EF Core to load them using the `ThenInclude` method. For example, we can load the publisher information for the editions of a book like so: 
+Now, what happens if one of the properties we include also has non-primitive type properties? By default, they won't be loaded from the database, but we can tell EF Core to load them using the `ThenInclude` method. For example, we can load the reader information for the checkout records of a book like so: 
  
 ```{.snippet} 
 using (var context = new LibraryContext()) 
 { 
 	var book = context.Book 
-		.Include(b => b.Editions) 
-			.ThenInclude(e => e.Publisher)
-		.Single(b => b.BookId == 1) ; 
+		.Include(b => b.CheckoutRecords) 
+			.ThenInclude(cr => cr.Reader)
+		.Single(b => b.Id == 1); 
 } 
 ``` 
 :::repl{data-name=then-include-method} 
 :::
  
-We can also chain `ThenInclude` calls to include deeper layers of related data. For example, if we want to find out the publisher information for the different editions of Fredrick Douglass' books in this library, we will need to chain `ThenInclude` calls to get the book, edition, and publisher information. 
+We can also chain `ThenInclude` calls to include deeper layers of related data. For example, if we want to find out the reader information for the checkout records of Fredrick Douglass' books in this library, we will need to chain `ThenInclude` calls to get the book, checkout records, and reader information. 
```

Comment:
My b on all of these wordings - I kinda just replaced the class names without registering how it sounded, and I definitely could have done better